cmd/state-handler/Dockerfile

```diff
@@ -4,10 +4,11 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o /bin/state-handler github.com/nmstate/kubernetes-nmstate/cmd/state-handler
 
 FROM centos:7
-RUN yum -y install epel-release && \
+RUN yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/nmstate/nmstate-el7/repo/epel-7/nmstate-nmstate-el7-epel-7.repo && \
+    yum -y install epel-release && \
     yum -y update && \
     yum -y install \
-        python2-pip \
+        nmstate \
         git \
         python2-pyyaml \
```

Comment:
The rest of those dependencies were needed because of the pip installation. They should be isntalled as a part of nmstate RPM now, we can drop them.

Reply:
ack, have drop them now, let's see what funct test says.